monkestation/code/modules/cassettes/cassette_approval.dm

```diff
@@ -121,6 +119,10 @@ GLOBAL_LIST_INIT(cassette_reviews, list())
 			to_chat(submitter, span_warning("You feel a wave of disapointment wash over you, you can tell that your cassette was denied by the Space Board of Music"))
 			logger.Log(LOG_CATEGORY_MUSIC, "[submitter]'s tape has been rejected by [usr]", list("approver" = usr.name, "submitter" = submitter.name))
 			action_taken = TRUE
+			verdict = "DENIED"
+
+/datum/cassette_review/ui_close()// Don't leave orphaned datums laying around. Hopefully this handles timeouts?
+	. = ..()
```

Comment:
did u forget to add a `qdel(src)` here or something?

Reply:
I just added this for a runtime thing I was trying to test. I don't think it is really needed since I am not actually doing anything special. Could just remove the whole thing. If I understand byond right its parent ui_close() should be called right?

Reply:
Ill add it just in case.